blocks/author-bio/author-bio.js

```diff
@@ -1,34 +1,73 @@
-import { createTag, getMetadata } from '../../scripts/scripts.js';
+import { createTag, getMetadata, lookupPeople } from '../../scripts/scripts.js';
 import authorsParser from '../../scripts/authors-parser.js';
 
-export default function decorate(block) {
+export default async function decorate(block) {
   const defaultAuthorImageSrc = '../../styles/images/AuthorPlaceholder.svg';
   const defaultAuthor = 'Merative';
 
-  let authors = getMetadata('authors');
+  let authors = [];
   const nameField = 'name';
   const titleField = 'title';
   const imageField = 'image';
   const content = [];
 
-  if (!authors) {
+  // get author details from people/query-index
+  const pathnames = [...block.querySelectorAll('a')].map((a) => {
+    const url = new URL(a.href);
+    if (url.hostname.endsWith('.page') || url.hostname.endsWith('.live') || url.hostname.endsWith('merative.com') || url.hostname.startsWith('localhost')) return url.pathname;
+    return a.href;
+  });
+
+  const peopleList = await lookupPeople(pathnames);
+  if (peopleList.length) {
+    peopleList.forEach((people) => {
+      const author = {};
+      let isAuthorFlag = false;
+
+      // Add name
+      if (people.title) {
+        author[nameField] = people.title;
+        isAuthorFlag = true;
+      }
+      // Add image
+      if (people.image !== '0' && people.title !== '0') {
+        const url = new URL(people.image, window.location.href);
+        author[imageField] = url.pathname;
+        isAuthorFlag = true;
+      }
+      // Add title
+      if (people['display-title']) {
```

Comment:
```suggestion
      if (people['display-title'] && people['display-title'] !== '0') {
```

Sometimes when there's no value added to the block row, the JSON query returns `0`. Let's add a check if `display-title` returns `0`, so that we don't have `0` rendered.